grafana-plugin/src/containers/PluginConfigPage/PluginConfigPage.tsx

```diff
@@ -129,25 +132,37 @@ export const PluginConfigPage = (props: Props) => {
   }, []);
 
   const handleSyncException = useCallback((e) => {
+    const buildErrMsg = (msg: string): string =>
+      constructSyncErrorMessage(msg, plugin.meta.jsonData.onCallApiUrl);
+
     if (plugin.meta.jsonData?.onCallApiUrl) {
-      let statusMessage = plugin.meta.jsonData.onCallApiUrl + '\n' + e + ', retry or check settings & re-initialize.';
-      if (e.response.status == 404) {
-        statusMessage += '\nIf Grafana OnCall was just installed, restart Grafana for OnCall routes to be available.';
+      const { status: statusCode  } = e.response;
+
+      let statusMessage: string;
+
+      if (statusCode == 403) {
+        statusMessage = buildErrMsg(INVALID_INVITE_TOKEN_ERROR_MSG);
+      } else if (statusCode === 404) {
+        statusMessage = buildErrMsg('If Grafana OnCall was just installed, restart Grafana for OnCall routes to be available.');
+      } else if (statusCode === 502) {
+        statusMessage = buildErrMsg(`Unable to communicate with either the Grafana API, or Grafana OnCall engine API. ${constructErrorActionMessage('verify that the API URLs that you entered are correct')}`);
+      } else {
+        statusMessage = buildErrMsg(`An unknown error occured. ${constructErrorActionMessage()}. If the error still occurs please reach out to support.`)
       }
       setPluginStatusMessage(statusMessage);
       setRetrySync(true);
     } else {
-      setPluginStatusMessage('OnCall has not been setup, configure & initialize below.');
+      setPluginStatusMessage(buildErrMsg('OnCall has not been setup, configure & initialize below.'));
     }
     setPluginStatusOk(false);
     setPluginConfigLoading(false);
   }, []);
 
-  const finishSync = useCallback((get_sync_response) => {
```

Comment:
renamed `get_sync_response` to `getSyncResponse` to make it consistent with camelCase used throughout the rest of the file